### ISSUE TITLE
Non-value CLI flags are not translated correctly

### ISSUE THREAD

**Issue by tvollstaedt** [OpenAPITools/openapi-generator-cli#113]
I've set up a generator config in `openapitools.json` with `"skipValidateSpec": true`, and just ran `openapi-generator-cli generate`. Unfortunately, this (and other non-value) flag is translated to `--skip-validate-spec=true`, but should be `--skip-validate-spec`. The client fails with:

```
[[angular] openapi_solve.yml] [error] Found unexpected parameters: [--skip-validate-spec=true]
[[angular] openapi_solve.yml] 
[[angular] openapi_solve.yml] See 'openapi-generator-cli help' for usage.
```

I'm not sure if this is a regression with the release of a new CLI version, but we should add exclusions in the generator for these flags.

**Comment by kay-schecker** [comment]
@tvollstaedt Thanks for reporting the issue. I'll look at it right away.

**Comment by kay-schecker** [comment]
fix released (Please note that v2.1.1 was not published to NPM, because I also did setup semantic release today).
Please use v2.1.3 instead.
